plugins/linux-drm/README.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Plugin: Linux DRM
+---
+
+## Introduction
+
+This plugin checks if there are monitors connected to each DRM device. The result will be used to
+inhibit devices that require a monitor to be attached.
+
+## External Interface Access
+
+This plugin requires read access to `/sys/class/drm`.
```

Comment:
@valentindavid can you please check if snap rules need to be updated for this new access?

Reply:
This is provided by interface `opengl`. But I am not sure we should connect to that. Maybe we should just add it to `fwupd` interface. I will bring it up.

Reply:
@superm1  Does it really need to read that directory, that is to list the symlinks that are there? Or does it need read access to devices of that subsystem? I feel it is porbably needs to be able to read `/sys/devices/**/drm` and `/sys/devices/**/drm/**`

Reply:
I believe you're right